index.html

```diff
@@ -0,0 +1,180 @@
+          </div>
+          <h3 class="p1title">Awards and Achievements</h3>
+          <p class="p1description">
+            Issue gifts to individuals who have contributed immensly towards development of holistic medicine in the
+            last year including journalists
+          </p>
+        </div>
+        <div class="p1">
+          <div class="p1icon">
+            <img class="icon-p" src="assets/pitch-icon.png" alt="" />
+          </div>
+          <h3 class="p1title">Presentations</h3>
+          <p class="p1description">
+            Have guest speakers give presentations of new developments as well as new inventions that may revolutionize
+            holistic medicine.
+          </p>
+        </div>
+        <div class="p1">
+          <div class="p1icon">
+            <img class="icon-p" src="assets/network-dark.png" alt="" />
+          </div>
+          <h3 class="p1title">Networking</h3>
+          <p class="p1description">
+            Attendees can mingle, exchange contact information and expand their networks further.
+          </p>
+        </div>
+
+        <button type="submit" class="join-button" id="join">
+          Join The movement and enjoy quality life.
+        </button>
+      </div>
+      <a id="program-link" href="#">See The Whole Program</a>
+    </section>
+
+    <section class="speakers-grid">
+      <h2 class="speakers-title">Featured Speakers</h2>
+      <hr class="hr">
+      <div class="speakers"></div>
+    </section>
+    <section class="partners" id="sponsor">
+      <h2 class="partners-title">Partners</h2>
+      <hr class="hr">
+      <div class="partner-items">
+        <div>
+          <a class="partner-link" href="https://www.gatesfoundation.org/" target="_blank" rel="noopener noreferrer"><img
+              class="gates" src="assets/BillMelindaGatesFoundationp.png" alt=""></a>
+        </div>
+        <div class="partner-container">
+          <a class="partner-link" href="https://www.opensocietyfoundations.org/" target="_blank"
+            rel="noopener noreferrer"><img class="open-s" src="assets/opensocietyfoundationsfitwidth.png" alt=""></a>
+            <a class="partner-link" href="https://www.who.int/" target="_blank" rel="noopener noreferrer"><img class="open-s"
+              src="assets/World_Health_Organizationfit.png" alt=""></a>
+              <a class="partner-link"
+            href="https://www.bloomberg.org/public-health/"
+            target="_blank"
+            rel="noopener noreferrer"
+            ><img class="bloomberg open-s" src="assets/PikPng.png" alt=""></a
+          >
+        </div>
+       
+      </div>
+    </section>
+    <footer id="contact">
+      <div><span>&copy;</span> 2022 Holistic Health annual Conference</div>
+      <div>All rights reserved.</div>
+      <div class="contacts">
+        <div class="organizer">
+          <p>Maribel Pauls</p>
+          maribelpaul@gmail.com
+        </div>
+      </div>
```

Comment:
- [ ] Kindly align the elements of the footer on the home page to match the original design, the logo on the left, and the text on the right!
_Your design_
![footer home](https://user-images.githubusercontent.com/47625407/175428909-f2fa385c-0433-437e-8f8a-690128b80163.png)
n